db/migrate/20230718173353_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :content
+      t.integer :author_id
+
+      t.timestamps
+    end
+    add_index :posts, :author_id
+  end
```

Comment:
- [ ] Kindly follow the ERD when setting fields for the models as required. In this file, you're missing out on the comments_counter and the likes_counter. Also, the post text field should have field title text as this will help to not have conflicts in other parts of the app while developing. Please follow the project guideline as in the ERD. 👍🏽

```ruby
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.timestamps 
```

<img width="323" alt="Screenshot 2023-07-18 at 6 41 13 PM" src="https://github.com/harshi0102/blog-app-rails/assets/50941074/866445a2-6ecb-4105-99b8-e935f3191018">